examples/gallery/3d_plots/scatter3d.py

```diff
@@ -36,9 +41,9 @@
 
 # Define a colormap to be used for three categories, define the range of the
 # new discrete CPT using series=(lowest_value, highest_value, interval),
-# use color_model="+c" to write the discrete color palette "cubhelix" in
-# categorical format
-pygmt.makecpt(cmap="cubhelix", color_model="+c", series=(0, 3, 1))
+# use color_model="+c" + species to write the discrete color palette "cubhelix" in
+# categorical format and add the species names extracted above as annotations
+pygmt.makecpt(cmap="cubhelix", color_model="+c" + species, series=(0, 2, 1))
```

Comment:
Since this is a tutorial for learning, it might be better to explicitly show that a categorical colour map is made using `color_model="+csetosa,versicolor,virginica"`. Also, might pay to check that this is the right order if you haven't already (i.e. that setosa is dark blue, versicolor is brown, virginica is light blue). I can't remember off the top of my head if this is correct.

Reply:
Modified it, the order is correct ;)